src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -189,9 +192,9 @@ impl Renderer for HtmlHandlebars {
         book.write_file("_FontAwesome/fonts/FontAwesome.ttf", theme::FONT_AWESOME_TTF)?;
 
         // Ace is a very large dependency, so only load it when requested
-        if book.uses_ace() {
+        if playpen_config.is_editable() {
             // Load the editor
-            let editor = editor::Editor::new(book.get_ace_path());
+            let editor = editor::Editor::new(playpen_config.get_editor());
             book.write_file("editor.js", &editor.js)?;
             book.write_file("ace.js", &editor.ace_js)?;
```

Comment:
>  I could call it `playpen_editor`, but this depends on what other editors mdBook plans to have, if any. Like, MathJax, is that another type of editor? Is this perhaps not the right time to worry about this?

Well you might be right, but from my perspective `playpen_editor` or even `playpen_editor_config` would be ideal as it would be easier for new contributors to orient around the code (Somehow even though I had some understanding of what is going on in the code the bare `editor` did not want to "click" for me until I've red all of this PR and finally understood that this is basically ACE config file)